lib/sycamore/sycamore/utils/pdf.py

```diff
@@ -108,14 +113,45 @@ def read_stderr(fh, q):
         t_err.join()
 
 
-def convert_from_path_streamed_batched(filename: str, batch_size: int) -> Generator[List[Image.Image], None, None]:
+def convert_from_path_streamed_batched(
+    filename: str, batch_size: int, hash_key: str, use_cache: bool = True
```

Comment:
I think as a function argument, a better name would be 'file_checksum' instead of 'hash_key'.  It just leaks internals.

Reply:
updated